projects/addon-editor/components/editor-socket/editor-socket.component.ts

```diff
@@ -18,6 +18,7 @@ import {TUI_SANITIZER} from '@taiga-ui/core';
 @Component({
```

Comment:
I wonder if we should downgrade it to `@Directive` and use `DirectiveStylesService` to handle those global styles. Just a TODO/TOTHINK :)